app/src/main/java/ch/epfl/qedit/model/MatrixFormat.java

```diff
@@ -0,0 +1,74 @@
+package ch.epfl.qedit.model;
+
+public class MatrixFormat extends AnswerFormat {
+    private Visitor visitor;
+    private boolean hasDecimal = true;
+    private boolean hasSign = true;
+
+    private int tableRowsNumber = 1;
+    private int tableColumnsNumber = 1;
+    private int maxCharacters = 3;
+    private String hintString = hint();
+
+    public MatrixFormat(
```

Comment:
I think we should move this class as an inner class of `AnswerFormat` for better encapsulation.

Reply:
I'm not sure it is a good idea to create inner classes with all formats. The more formats we have and the less readable the AnswerFormat class would be.

Reply:
Yeah, that's right as well, so I think it is fine like this